pydantic_errors_messages_translations/__init__.py,sha256=ykyivpwz4EaZr1FiJArbpYB1ds_lpml_RVAs0Sex5JE,152
pydantic_errors_messages_translations/helpers.py,sha256=kvRVKIFdPJ9a8jiSMI78YfWHmsc8O83BpBiBOt_myL0,391
pydantic_errors_messages_translations/translations.py,sha256=GrJRkVxyRjd8yPk5IQA5sYk68W6taeUTwEC-b9WEyJ4,6934
pydantic_errors_messages_translations-1.1.1.dist-info/METADATA,sha256=RMZIFDtb6Lk5L5YKLQVhsJgeTWE7-7Q6qeVyhwbh18Y,1735
pydantic_errors_messages_translations-1.1.1.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
pydantic_errors_messages_translations-1.1.1.dist-info/top_level.txt,sha256=F731NCWFoccx8UUMdOVBBr2yEJGlISUHSiiwzMzHx6Q,38
pydantic_errors_messages_translations-1.1.1.dist-info/RECORD,,
